lib/src/widgets/login_body.dart

```diff
@@ -0,0 +1,256 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '../constants/constants_color.dart';
+import '../blocs/login_blocs/login_blocs.dart';
+import '../blocs/login_blocs/login_event.dart';
+import '../blocs/login_blocs/login_states.dart';
+import '../screens/home_screen.dart';
+import '../utils/validators.dart';
+import '../repositories/user_repository.dart';
+import '../constants/constants.dart';
+
+class Login_Body extends StatefulWidget {
```

Comment:
can u tell me login body mean?

Reply:
That is name of part 
![image](https://user-images.githubusercontent.com/33689489/127883915-bf678575-5652-4dff-a799-be9a7cbdcc2f.png)

I think it is body of login